components/training/presenterBlock.tsx

```diff
@@ -0,0 +1,168 @@
+import Image from "next/image";
+import type { Template } from "tinacms";
+import { tinaField } from "tinacms/dist/react";
+import { TinaMarkdown } from "tinacms/dist/rich-text";
+import { presenterSchemaConstants } from "../../.tina/collections/presenter";
+import { Container } from "../util/container";
+
+export const PresenterBlock = ({ data }) => {
+  return (
+    <Container size="custom">
+      <h2
+        className="mb-12 text-center"
+        data-tina-field={tinaField(data, presenterBlockConstant.header)}
+      >
+        About our <span className="text-sswRed">{data.header}</span>
+      </h2>
+      <div className="grid gap-6 md:grid-cols-3">
+        {data.presenterList?.map((p, i) => (
+          <PresenterCard
+            key={i}
+            presenter={p.presenter}
+            schema={data.presenterList}
+            index={i}
+          />
+        ))}
+      </div>
+      {data.otherEvent && (
+        <div className="grid grid-cols-12 px-4 py-12 text-center text-xl">
+          <div className="col-span-12 justify-center md:inline-flex">
+            <span className="self-center">You can also check our</span>
+            <span
+              className="flex justify-center self-center"
+              data-tina-field={tinaField(
+                data.otherEvent,
+                presenterBlockConstant.otherEvent.title
+              )}
+            >
+              <a
+                href={data.otherEvent.eventURL}
+                className="inline-flex cursor-pointer !no-underline hover:!underline"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <span className="p-2">{data.otherEvent.title}</span>
+              </a>
+            </span>
+          </div>
+        </div>
+      )}
+      {!data.otherEvent && <div className="pb-12"></div>}
+    </Container>
+  );
+};
+
+const PresenterCard = ({ presenter, schema, index }) => {
+  return (
+    <div
+      className="flex h-fit flex-col rounded-md border-b-4 border-b-sswRed bg-gray-100 p-10 text-center text-xl shadow drop-shadow md:h-full"
```

Comment:
The cards shouldn't have a drop shadow - everything else is looking great!

Reply:
I removed the drop-shadow from the div.